docs/website/docusaurus-odo-plugin-segment/README.md

```diff
@@ -0,0 +1,22 @@
+This plugin has been derived from https://github.com/xer0x/docusaurus-plugin-segment.
```

Comment:
Does it really make sense to have this doc here? If yes, we will have three places where our docs are scattered. One is odo.dev, other is odo wiki and third will be this. For things related to odo development, I would prefer to keep them on odo wiki, not even on odo.dev.

Reply:
It is a README for the plugin itself. Even though it is a part of odo development, I believe this is the best place to keep this information. In the best case, no one would need to bother changing this plugin, but for anyone who wants to, the plugin's README is probably the most intuitive place to find information related to the plugin.

Reply:
If it's the plugin README, the best place to find it is on the plugin's homepage, IMO. What good is it to have its README in odo repo? If anything, we can document what place to look at or how to tweak things if there's a problem with the plugin. Otherwise, I don't see why we should have it on odo repo.

Reply:
I do not understand what you mean by _the plugin's homepage_? This is a custom plugin written specifically for odo even though a lot of its content is derived from another plugin. odo repo is the only place it has been defined at. I am still not convinced that this should be moved to a different place.

Reply:
I had been missing all the context here. Now it makes more sense.

Reply:
Sorry about that, I'll clarify a few more things in the README.